src/test/testing_assert.hpp

```diff
@@ -64,6 +64,27 @@ bool contained_in_query_plan(const std::shared_ptr<const AbstractOperator>& node
   EXPECT_TABLE_EQ(opossum_table, expected_table, OrderSensitivity::Yes, TypeCmpMode::Strict, \
                   FloatComparisonMode::AbsoluteDifference)
 
+/**
+ * Compare two segments with respect to OrderSensitivity, TypeCmpMode and FloatComparisonMode
+ */
+#define EXPECT_SEGMENT_EQ(segment_to_test, expected_segment, order_sensitivity, type_cmp_mode, float_comparison_mode) \
+  EXPECT_TRUE(segment_to_test&& expected_segment&& check_segment_equal(                                               \
```

Comment:
Looks like the formatter went berserk here

Reply:
It wouldn't bother me. Shall I add the two lines for clang-format to disable it or leave it as is?

Reply:
It took me some seconds to understand what happens here. I'd add those two lines.

Reply:
Wait? You are not talking about the missing 10 spaces for the third line but about something else, right? What did confuse you exactly?

Reply:
The `&&` that looks like a universal reference but is an "and"